src/components/iconButton/styles.module.css

```diff
@@ -0,0 +1,7 @@
+.iconButton {
+    border: none;
+    outline: none;
+    box-shadow: none;
+    background-color: var(--page-bg);
+    transition: background-color .5s ease-in-out, color .5s ease-in-out;
```

Comment:
use same time-duration throughout, at other places I have seen 0.25s and 0.2s, change all of them to same value

Reply:
Now, I am using 0.5s everywhere.